doc/nrf/ug_bt_mesh.rst

```diff
@@ -0,0 +1,91 @@
+the Bluetooth Low Energy protocol to exchange messages between the nodes on the network.
+All nodes in a Bluetooth Mesh network can communicate with each other, as long as there's a chain of nodes between them to relay the messages.
+The messages are encrypted with two layers of 128 bit AES-CCM encryption, allowing secure communication between thousands of devices.
+
+The end-user applications are implemented as a set of Mesh Models.
+The Bluetooth SIG defines some generic and reusable models in the `Mesh Model Specification`_, but vendors are free to define their own models.
+
+Read more about the Bluetooth Mesh specification on the Bluetooth SIG website:
+
+* `Bluetooth Mesh Glossary`_
+* `Bluetooth Mesh FAQ`_
+* `Bluetooth Mesh Study Guide`_
+
+.. _mesh_ug_supported features:
+
+Supported features
+******************
+
+The Bluetooth Mesh in |NCS| supports all mandatory features of the `Bluetooth Mesh Profile Specification`_ and the `Mesh Model Specification`_, as well as most of the optional features.
+
+The following features are supported by Zephyr's :ref:`zephyr:bluetooth_mesh`:
+
+* :ref:`zephyr:bluetooth_mesh_core`:
+
+  * Node role
+  * Relay feature
+  * Low power feature
+  * Friend feature
+
+* :ref:`zephyr:bluetooth_mesh_provisioning`:
+
+  * Provisionee role (GATT and Advertising Provisioning bearer)
+  * Provisioner role (only Advertising Provisioning bearer)
+
+* :ref:`GATT Proxy Server role <zephyr:bt_mesh_proxy>`
+* :ref:`zephyr:bluetooth_mesh_models`
+
+In addition to these features, |NCS| implements several models defined in the  `Mesh Model Specification`_, see :ref:`bt_mesh_models` for details.
+
+Configuration
+*************
+
+The Bluetooth Mesh support is controlled by :option:`CONFIG_BT_MESH`, which depends on the following configuration options:
+
+* :option:`CONFIG_BT`
+* :option:`CONFIG_BT_OBSERVER`
+* :option:`CONFIG_BT_PERIPHERAL`
+
+The Bluetooth Mesh subsystem currently performs best with :ref:`Zephyr's Bluetooth LE Controller <zephyr:bluetooth-arch>`.
+To force |NCS| to build with this controller, enable :option:`CONFIG_BT_LL_SW_SPLIT`.
+
+
+Optional configuration options
+==============================
+
+Optional features in the Bluetooth Mesh stack must be explicitly enabled:
+
+* :option:`CONFIG_BT_MESH_RELAY`
+* :option:`CONFIG_BT_MESH_FRIEND`
+* :option:`CONFIG_BT_MESH_LOW_POWER`
+* :option:`CONFIG_BT_MESH_PROVISIONER`
+* :option:`CONFIG_BT_MESH_GATT_PROXY`
+* :option:`CONFIG_BT_MESH_PB_GATT`
+* :option:`CONFIG_BT_MESH_CDB`
+
+The |NCS| Bluetooth Mesh Model implementations are also optional features, and must be explicitly enabled to be included in the build.
+See :ref:`bt_mesh_models` for details.
+
+Persistent storage of the Bluetooth Mesh provisioning and configuration data is enabled by :option:`CONFIG_BT_SETTINGS`.
+See the Persistent Storage section of :ref:`zephyr:bluetooth-arch` for details.
+
+
+Available drivers, libraries, and samples
```

Comment:
Are there any tools Mesh is using? nRF Mesh app, for example? We could include these in a "Mesh tools" section.